sdk/go/common/util/cmdutil/trace.go

```diff
@@ -37,7 +36,7 @@ import (
 // TracingEndpoint is the Zipkin-compatible tracing endpoint where tracing data will be sent.
 var TracingEndpoint string
 
-// Deprecated. TracingToFile=true if pulumi was called with a file://
+// TracingToFile Deprecated. TracingToFile=true if pulumi was called with a file://
```

Comment:
Looks like Go style is to put "Deprecated:" at the beginning of a paragraph in the doc comment:

```go

// TracingToFile indicates if pulumi was called with a file:// scheme URL (--tracing=file:///...). 
//
// Deprecated: Even in this case TracingEndpoint will now have the tcp:// scheme and will point to a
// proxy server that will append traces to the user-specified file. Plugins should respect
// TracingEndpoint and ignore TracingToFile.
```

Reply:
Sorry for my mistake, I'll fix this later. Thanks for your correction.